packages/wonder-blocks-tokens/src/tokens/font.ts

```diff
@@ -1,30 +1,46 @@
+import {sizing} from "./sizing";
+
+export const fontFamily = {
+    sans: 'Lato, "Noto Sans", sans-serif',
+    serif: '"Noto Serif", serif',
+    mono: "Inconsolata, monospace",
+};
+
+export const fontWeight = {
+    light: 300,
+    regular: 400,
+    bold: 700,
+    black: 900,
+};
+
 export const font = {
     family: {
-        sans: 'Lato, "Noto Sans", sans-serif',
-        serif: '"Noto Serif", serif',
-        mono: "Inconsolata, monospace",
+        sans: fontFamily.sans,
+        serif: fontFamily.serif,
+        mono: fontFamily.mono,
     },
     size: {
-        xxxLarge: 36,
-        xxLarge: 28,
-        xLarge: 24,
-        large: 20,
-        medium: 16,
-        small: 14,
-        xSmall: 12,
+        xxxLarge: sizing.size_360,
+        xxLarge: sizing.size_280,
+        xLarge: sizing.size_240,
+        large: sizing.size_200,
+        medium: sizing.size_160,
```

Comment:
This one will need to change in a TB theme file: Foundations uses `sizing.size_180` for `medium`: https://www.figma.com/design/9seilz0fBgo3QUj6eXM0yR/%E2%9A%A1%EF%B8%8F-Foundations?node-id=52-3&t=NFXovVlCFbYhQHvm-0

Reply:
Agreed! you might want to look at the CSS vars refactor that I've been doing + the TB theming PR, as probably most of this stuff will get moved to the `theme/` folder to convert these tokens as CSS variables + create the `tb` version of these tokens. We can chat later about this!